src/ansys/meshing/prime/examples/examples.py

```diff
@@ -1453,3 +1459,108 @@ def download_block_model_pmdat(
 
     """
     return get_file(Examples.BLOCK_MODEL_PMDAT, destination, force)
+
+
+def download_wheel_ground_scdoc(
+    destination: Optional[str] = None, force: bool = False
+) -> Union[str, os.PathLike]:
+    """Download SCDOC file for the wheel ground contact patch example.
+
+    Parameters
+    ----------
+    destination: Optional[str]
+        Destination for the file to be downloaded.
+        If nothing is provided, the default path in app data is used.
+    force: bool
+        Option to download the file.
+        If true, the file is always downloaded.
+        If false, an existing file in the cache may be reused.
```

Comment:
```suggestion
    force: bool, optional
        Whether to always download the file. The default is ``False``, in which
        case a cached file is reused.
```
I notice that you aren't using what we consider the standard format, placing either "optional" after the data type and the default in the description. The other standard is to simply place the default after the data type like this: bool, default: False. I am not going to suggest additional edits to parameter descriptions but wanted to point out how your library differs from others.

Reply:
@PipKat yes, unintended.  This should be resolved now.